spec/rails_helper.rb

```diff
@@ -27,7 +27,13 @@
 # directory. Alternatively, in the individual `*_spec.rb` files, manually
 # require only the support files necessary.
 #
-Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
```

Comment:
are we sure only need two directories to include?

Reply:
It was because of dependency b/w `validate_helm_values_of_matcher` and `string_helper`. We pushed a fix for it.